master/include/comm/communicator.hpp

```diff
@@ -158,26 +159,30 @@ inline void Communicator::parse_message(const CAN_message_t& msg) {
 }
 
 inline int Communicator::publish_state(const int state_id) {
-    const uint8_t msg[] = {static_cast<unsigned char>(state_id)};
-
-    send_message(1, msg, STATE_MSG);
+    const uint8_t msg[] = {STATE_MSG, static_cast<uint8_t>(state_id)};
 
+    send_message(2, msg, MASTER_ID);
     return 0;
 }
 
 inline int Communicator::publish_mission(int mission_id) {
-    const uint8_t msg[] = {static_cast<unsigned char>(mission_id)};
-
-    send_message(1, msg, MISSION_MSG);
+    const uint8_t msg[] = {MISSION_MSG, static_cast<uint8_t>(mission_id)};
 
+    send_message(2, msg, MASTER_ID);
     return 0;
 }
 
 inline int Communicator::publish_left_wheel_rpm(double value) {
```

Comment:
It might be interesting to publish them both together: the AS will only use them both together and this could be a way of synchronizing the two